services/httpd/service.go

```diff
@@ -68,7 +69,8 @@ func NewService(c Config, hostname string, l *log.Logger, li logging.Interface)
 			li,
 			c.SharedSecret,
 		),
-		logger: l,
+		logger:           l,
+		httpServerLogger: li.NewStaticLevelLogger("[go-http]", log.LstdFlags, logging.ERROR),
```

Comment:
I was thinking that originally, but I wanted there to be a distinction between there being an error in the `httpd` service and there being an error in the http server.

The more I think of it,`httpd` is better.